Tools/GNUMake/Make.defs

```diff
@@ -814,14 +814,9 @@ ifeq ($(USE_UPCXX),TRUE)
   include        $(AMREX_HOME)/Tools/GNUMake/Make.upcxx
 endif
 
-ifeq ($(USE_SUNDIALS_3x4x),TRUE)
-  ifeq ("$(SUNDIALS_VERSION)","4")
-    $(info Loading $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials4...)
-    include        $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials4
-  else
-    $(info Loading $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials3...)
-    include        $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials3
-  endif
+ifeq ($(USE_SUNDIALS),TRUE)
+  $(info Loading $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials...)
+  include        $(AMREX_HOME)/Tools/GNUMake/packages/Make.sundials
 endif
 
 ifeq ($(USE_CVODE),TRUE)
```

Comment:
It looks like Tools/GNUMake/packages/Make.cvode was deleted, so code which relies on USE_CVODE=TRUE will fail. The initial purpose of having two separate flags for USE_SUNDIALS and USE_CVODE was to allow backward compatibility, particularly for users who rely on the Cray TPSL (cori's current version of that is 18.6, which I believe corresponds to cvode 2.7).

Reply:
which apps are depending on the Crap TPSL?  What if we asked that it be updated to more recent versions of CVODE/SUNDIALS?  I don't know the issues involved.

Reply:
I have added the cvode 2.7 related pieces back in.